### ISSUE TITLE
Quarkus IT error: HealthDistTest

### ISSUE THREAD

**Issue by miquelsi** [keycloak/keycloak#23965]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

ci

### Describe the bug

This error has happened in several workflows and in several jobs, so does not look like a flaky/temporary issue:

[Quarkus IT (ubuntu-latest, zip)](https://github.com/keycloak/keycloak/actions/runs/6502311810/job/17661502967)
[Quarkus IT (ubuntu-latest, container)](https://github.com/keycloak/keycloak/actions/runs/6502311810/job/17661503202)
[Quarkus IT (ubuntu-latest, container)](https://github.com/keycloak/keycloak/actions/runs/6505110185/job/17668474714)

```
Error: OR] Errors: 
Error: OR]   HealthDistTest.testBlockingProbes:80 » Connect Connection refused
Error: OR]   HealthDistTest.testNonBlockingProbes:69 » Connect Connection refused
```

```
Error: Exception in thread "pool-25-thread-8" java.lang.RuntimeException: Failed to stop and remove container
	at org.keycloak.it.utils.DockerKeycloakDistribution$1.run(DockerKeycloakDistribution.java:180)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: com.github.dockerjava.api.exception.NotFoundException: Status 404: {"message":"Cannot kill container: d3e6f60aa98ce8eb7b6fad553be14940c9e4ddefd331e06a8c822a04cbde0167: No such container: d3e6f60aa98ce8eb7b6fad553be14940c9e4ddefd331e06a8c822a04cbde0167"}
```

### Version

latest

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

**Comment by ahus1** [comment]
@mabartos - I think I found the problem as I didn't split the parameters in the previous test I added. Unsure why it worked before, and now stopped working. Picking it up, creating a PR later today.

**Comment by ahus1** [comment]
PR available for review: #23979

**Comment by mabartos** [comment]
@ahus1 Thanks, Alex!
